crates/fluvio-cluster/src/start/local.rs

```diff
@@ -149,25 +148,6 @@ pub struct LocalConfig {
     #[builder(setter(into, strip_option), default)]
     chart_location: Option<UserChartLocation>,
 
-    /// Whether to install the `fluvio-sys` chart in the full installation.
-    ///
-    /// Defaults to `true`.
-    ///
-    /// # Example
-    ///
-    /// If you want to disable installing the system chart, you can do this
-    ///
-    /// ```
-    /// # use fluvio_cluster::{ClusterError, LocalConfigBuilder};
-    /// # fn example(builder: &mut LocalConfigBuilder) -> Result<(), ClusterError> {
-    /// let config = builder
-    ///     .install_sys(false)
-    ///     .build()?;
-    /// # Ok(())
-    /// # }
-    /// ```
-    #[builder(default = "true")]
-    install_sys: bool,
```

Comment:
I think that this is not needed for `--local`. It is not being used

Reply:
k